### DISCUSSION TITLE
After installing NvChad, nothing happens when I open Neovim

### DISCUSSION THREAD

**Discussion by refaatcrafts** [NvChad/NvChad#2933] [Q&A] [answered]
After installing the NvChad **base** installation, nothing happens when I open Neovim. The plugins do not begin to install, and the configurations are not applied.
the installation done using this command
```sh
git clone https://github.com/NvChad/NvChad ~/.config/nvim --depth 1
```
and here is the file tree for more info
.
├── LICENSE
└── lua
    └── nvchad
        ├── autocmds.lua
        ├── configs
        │   ├── cmp.lua
        │   ├── gitsigns.lua
        │   ├── lazy_nvim.lua
        │   ├── lspconfig.lua
        │   ├── luasnip.lua
        │   ├── mason.lua
        │   ├── nvimtree.lua
        │   ├── telescope.lua
        │   └── treesitter.lua
        ├── mappings.lua
        ├── options.lua
        └── plugins
            └── init.lua

**Comment by siduck** [comment, marked as answer]
clone the starter repo! didnt you read nvchad.com install docs
